jets/c/ripn.c

```diff
@@ -0,0 +1,167 @@
+/* j/c/ripn.c
+**
+*/
+#include "all.h"
+
+#define MIN(x,y) ((x)>(y) ? (y) : (x))
+
+#define DROPBITS(n,w)      ((w) >> (n))
+#define TAKEBITS(n,w)      ((w) & ((1 << (n)) - 1))
+#define BITSLICE(n,off,w)  TAKEBITS(n, DROPBITS(32-(off+n), (w)))
+
+#define MERGE(n,ins,sig)  ((ins) | (sig) << n)
```

Comment:
Looks like of these macros, only `MIN` and `TAKEBITS` are used. I think we should delete the others, unless there's some reason not to.